doc/source/getting-started-mac-tutorial.rst

```diff
@@ -0,0 +1,155 @@
+    If necessary create a mac directory under the symbols directory (this will become unnecessary in future versions).
+
+
+Listing plugins
+---------------
+
+The following is a sample of the macOS plugins available for volatility3, it is not complete and more more plugins may
+be added.  For a complete reference, please see the volatility 3 :doc:`list of plugins <volatility3.plugins>`.
+For plugin requests, please create an issue with a description of the requested plugin.
+
+.. code-block:: shell-session
+
+    $ python3 vol.py --help | grep -i mac. | head -n 4
+        mac.bash.Bash       Recovers bash command history from memory.
+        mac.check_syscall.Check_syscall
+        mac.check_sysctl.Check_sysctl
+        mac.check_trap_table.Check_trap_table
+
+.. note:: Here the the command is piped to grep and head in-order to provide the start of the list of macOS plugins.
+
+
+Using plugins
+-------------
+
+The following is the syntax to run the volatility CLI.
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f <path to memory image> <plugin_name> <plugin_option>
+
+
+Example
+-------
+
+banners
+~~~~~~~
+
+In this example we will be using a memory dump from the Securinets CTF Quals 2019 Challenge called Contact_me.  We will limit the discussion to memory forensics with volatility 3 and not extend it to other parts of the challenge.
+Thanks go to `stuxnet <https://github.com/stuxnet999/>`_ for providing this memory dump and `writeup <https://stuxnet999.github.io/securinets-ctf/2019/08/24/SecurinetsQuals2019-Contact-Me.html>`_.
+
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f contact_me banners.Banners
+        
+        Volatility 3 Framework 2.4.2
+
+        Progress:  100.00               PDB scanning finished
+        Offset  Banner
+        
+        0x4d2c7d0       Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64
+        0xb42b180       Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64
+        0xcda9100       Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64
+        0x1275e7d0      Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64
+        0x1284fba4      Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64
+        0x34ad0180      Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64
+        
+
+The above command helps us to find the memory dump's Darwin kernel version. Now using the above banner we can search for the needed ISF file.
+If ISF file cannot be found then, follow the instructions on :ref:`getting-started-macos-tutorial:Procedure to create symbol tables for macOS`. After that, place the ISF file under the ``volatility3/symbols/mac`` directory.
```

Comment:
This can now just say `volatility3/symbols`

Reply:
All right. I've corrected the path.